keyboards/dztech/dz65rgb/v3/rules.mk

```diff
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = lufa-ms
```

Comment:
Again, I think this is a bad idea, and is very dangerous. 

Any files that are added to the flash drive created that isn't the flash.bin file *will* brick the device. 

This is super common on Windows and macOS.

You're using a bootloader that WILL lead to bricked boards.

Reply:
Thank you for ur suggestion, actually this is not happend during the test, but the  default bootloader often cause "not blank at 0x01" when flashing, and they often put the DFU driver into normal mode.

Reply:
Yeah, but my concern is .... we've seen a lot of issues with boards like the tada68, which uses the mass storage bootloader.  Not just theoretically, but you can find a lot of posts about these boards bricking... basically on a whim. 

Another issue, is that the fuses for the ATmega32u4 only protect up to 2048 words (4k bytes).  This means that anything past that is not protected.  The mass storage bootloader is 6-8K, depending on the version of LUFA that you're using to compile with.

As for the "not blank at 0x01" issue, yeah, that does happen.  Usually ISP flashing will fix that.  which is what you're trying to avoid.  But I'm certain that you will see MORE issues using this bootloader. 

You'd probably be better off with the HID Bootloader, and would actually gain you 2K of firmware space.